src/utils/gauth.js

```diff
@@ -0,0 +1,96 @@
+import Axios from 'axios';
+import store from '@/store';
+
+const clientId = '746645565897-flmj7tj1hu754tl4uul2do8cq1sslp27.apps.googleusercontent.com';
+const clientSecret = 'vP98OJ0nebq8qbeoBzKT5HvG';
```

Comment:
I don't think you want that in your commit

Reply:
It's a test account so no problems but for now the only way I found for it to be firefox and chrome compatible is to use the clientSecret to get the refresh_token otherwise the google login page opens too often